tests/db_tests/test_api.py

```diff
@@ -58,13 +59,17 @@ def try_to_mount_device():
 
     # mock output of actual mounting command
     def mount(path):
-        return "Mounted"
+        return "Mounted at /dummy/path"
```

Comment:
Unrelated to this PR, but it popped to my mind when reading this line:

Since recently I only get
```
successfully mounted device at:
```

with a missing path, whenever my device gets mounted. Could this be related to the `rstrip` you added [here](https://github.com/fgebhart/workoutizer/pull/184/files#diff-9071e041a6adf4f8168e822fd9b01a8498f2be2caf6efdc94f0dccd12d21e2cfR115)? 

Just didn't have the time to look into this in detail.